src/App.js

```diff
@@ -1,19 +1,23 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { Switch, Route } from 'react-router-dom';
+import Header from 'components/header';
+import Home from 'components/home';
+import Favorites from 'components/favorites';
+import { Provider } from 'react-redux';
+import store from 'store';
 
 class App extends Component {
   render() {
     return (
-      <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h1 className="App-title">Welcome to React</h1>
-        </header>
-        <p className="App-intro">
-          To get started, edit <code>src/App.js</code> and save to reload.
-        </p>
-      </div>
+      <Provider store={store}>
+        <div className="App">
+          <Header />
+          <Switch>
+            <Route exact path="/" component={Home} />
+            <Route path="/favorites" component={Favorites} />
+          </Switch>
```

Comment:
It would be nice to handle not found routes and either redirect the user to a known route or to a specific component letting him know what happened

Reply:
Nice one, I just updated it to include the Redirect, thanks 😃